lib/service/task_service.dart

```diff
@@ -8,8 +8,9 @@ class TaskService {
 
   TaskService({BackendClient? client}) : client = client ?? BackendClient();
 
-  Future<void> addTask({required Task task}) async {
+  Future<void> addTask({required Task task, TaskStatus? status}) async {
```

Comment:
Please remove the parameter `status`. It is not used.

Reply:
done